datalake/src/main/java/com/sequenceiq/datalake/service/resize/recovery/ResizeRecoveryService.java

```diff
@@ -70,6 +70,12 @@ public SdxRecoverableResponse validateRecovery(SdxCluster sdxCluster) {
         switch (actualStatusForSdx.getStatus()) {
             case STOP_FAILED:
                 return new SdxRecoverableResponse("Resize can be recovered from a failed stop", RecoveryStatus.RECOVERABLE);
+            case STOPPED:
+                if (sdxCluster.isDetached()) {
+                    return new SdxRecoverableResponse("Resize can reattach cluster", RecoveryStatus.RECOVERABLE);
```

Comment:
Until we test it and are sure of it, let's call this non-recoverable state.

When the changes for [CB-14399](https://jira.cloudera.com/browse/CB-14399) land. We can very the behavior and make changes accordingly.